engo.go

```diff
@@ -5,14 +5,20 @@ import (
 	"log"
 
 	"engo.io/ecs"
+	"engo.io/engo/act"
 )
 
 var (
 	// Time is the active FPS counter
 	Time *Clock
 
 	// Input handles all input: mouse, keyboard and touch
-	Input *InputManager
+	Input *InputMgr
```

Comment:
I'm not a fan of this sort of shortening. I'd strongly prefer having `InputManager`.